src/main.rs

```diff
@@ -8,6 +8,8 @@ use std::process::exit;
 
 use toml::Value;
 
+type Result<T, E = String> = std::result::Result<T, E>;
```

Comment:
That's not a great idea because `Result` is already in the current scope. Can you change the alias name please?

Reply:
That's intended, with `E` defaulted instead of the usual `type Result<T> = std::result::Result<T, String>;` that does not allow later overriding. In other words, this is overriding the `Result` in scope with one that has a sensible default for `E`.

Reply:
Fair enough!